src/http_module.cpp

```diff
@@ -67,6 +69,12 @@ ngx_command_t gCommands[] = {
       ngx_conf_set_str_slot,
       NGX_HTTP_MAIN_CONF_OFFSET,
       offsetof(MainConf, serviceName) },
+    
+    { ngx_string("otel_resource_attr"),
```

Comment:
The idea was to call the block "otel_resource" and, perhaps, add some other resource related options to it later on (or even support multiple resources).

"otel_resource_attr" name makes more sense for single attribute directive, like `otel_resource_attr name value;`. It's a reasonable approach, but then it shouldn't be a block.